teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<p>Is the number for the 'next' notebook already fixed? If yes, please include it here.</p><p>... refer to <strong>Txxx</strong> ... (maybe even with a link if available)</p><p> </p><p>At the very and, we usually have a small quiz (three questions) can you think of any?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>The number is T020.</p><p>We could link to the right folder in GitHub or do we wait for the website?</p>